ITEventsTests/DateFormatterServiceTests.swift

```diff
@@ -0,0 +1,53 @@
+import XCTest
+@testable import ITEvents
+
+class DateFormatterServiceTests: XCTestCase {
+    var dateFormatterService: DateFormatterService!
```

Comment:
All your code in tests should work also with a protocol not with implementation:
```swfit
var dateFormatterService: IDateFormatterService!
```